deno_dist/middleware/csrf/index.ts

```diff
@@ -11,6 +11,43 @@ const isSafeMethodRe = /^(GET|HEAD)$/
 const isRequestedByFormElementRe =
   /^\b(application\/x-www-form-urlencoded|multipart\/form-data|text\/plain)\b/
 
+/**
+ * CSRF protection middleware for Hono.
+ *
+ * @see {@link https://hono.dev/middleware/builtin/csrf}
+ *
+ * @param {CSRFOptions} [options] - The options for the CSRF protection middleware.
+ * @param {string|string[]|IsAllowedOriginHandler} [options.origin] - Specify origins.
```

Comment:
Writing the following instead of `IsAllowedOriginHandler` would be more helpful.

```ts
(origin: string, context: Context) => boolean
```

Reply:
Thanks. Resolved and squashed to first commit.